src/util/forms/FormCheckboxTableCell.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @author psarando
+ */
+import Checkbox from "@material-ui/core/Checkbox";
+import TableCell from "@material-ui/core/TableCell";
+
+// Apparently only necessary for FastField, but maybe more correct for "vanilla" Field as well.
+const onCheckboxChange = (setFieldValue, fieldName) => (event, checked) => {
+    setFieldValue(fieldName, checked);
+};
```

Comment:
Why not import this from `FormCheckbox`?

Reply:
Oh, whoops. Probably could do that (if I have FormCheckbox export it, anyway)